### DISCUSSION TITLE
GPT4ALL models

### DISCUSSION THREAD

**Discussion by misza80** [mckaywrigley/chatbot-ui#1598] [Help]
Hi, is it possible to incorporate other local models with chatbot-ui, for example ones downloaded from gpt4all site, likke gpt4all-falcon-newbpe-q4_0.gguf model?

**Comment by Trapper4888** [comment]
Yes, check out ollama integration.

**Reply by misza80** [reply]
Nice one, will chek it out. Thank  you.

**Reply by misza80** [reply]
After setting up Nous-Hermes-2-Mistral-7B-DPO.Q4_0.gguf using Import functionality in ollama.
Thanks again for your suggestion.

**Comment by Jonneal3** [comment]
You can really use any model you want --- Custom models are with openrouter.com - check that out! Every model i can think of is on there

**Reply by misza80** [reply]
Will check out this site, thanks.
